docs/debug.md

```diff
@@ -32,9 +32,9 @@ space of an entirely different process; and that lasts no more than a few second
    debugging. We need to stop this provider process from exiting before we can connect. So, put this bit of code in place:
 ```go
 	connected := false
-    for !connected {
-        time.Sleep(time.Second) // set breakpoint here
-    }
+for !connected {
```

Comment:
I mean, this is how you can debug a live instance but it's totally unnecessary - drive it with tests

Reply:
This is auto-generated doc.